src/html2ans/parsers/text.py

```diff
@@ -185,9 +185,12 @@ def parse(self, element, *args, **kwargs):
                             content_elements.append(p_result.output)
             else:
                 p_result = ParagraphParser().parse(element)
+                if p_result.match and p_result.output:
+                    content_elements.append(p_result.output)
                 if p_result.match:
```

Comment:
Won't this adding the output twice if it exists?

Reply:
Yup, forgot to remove it

Reply:
Hmm can we have a test so that it would have failed in that case? 😬